tests/opts.rs

```diff
@@ -393,22 +393,6 @@ fn issue_665() {
     assert_eq!(res.unwrap_err().kind, ErrorKind::EmptyValue);
 }
 
-#[test]
-fn issue_1047_min_zero_vals_default_val() {
```

Comment:
Why is this deleted?

Reply:
This test is a regression test for an issue #1047 which directly asks for the default value to be used when the option is passed without a value.

As we now have `default_missing_value` for this case, this PR has deliberately broken this behaviour and so the use is no longer is supported.

If you like, I can change this test to use `default_missing_value` instead of deleting it?

Reply:
Yes please.